modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -33,10 +33,11 @@ import AposThemeMixin from 'Modules/@apostrophecms/ui/mixins/AposThemeMixin';
 export default {
   name: 'TheAposAdminBar',
   mixins: [ AposThemeMixin ],
-  data() {
```

Comment:
Is this no longer a thing? Wondering if this change was strictly necessary or not